ecom_test/tests/backend_tests/products/test_list_products_with_filter.py

```diff
@@ -0,0 +1,40 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.dao.products_dao import ProductsDAO
+from datetime import datetime, timedelta
+
+
+@pytest.mark.regression
+class TestListProductsWithFilter(object):
+
+    # create a method to get the list of products with filter for specific period of time from now
+
+    @pytest.mark.tcid35
```

Comment:
I am assuming "tcid35" is your own. So it is difficult for me to match it with mine. Incase I want to compare the code to mine.
I think your tcid35 is same as mine tcid51. Can you may be add another mark (may be @pytest.mark.ssqaid51. We can discuss this further if you like.

Reply:
it's different from yours